src/ripple/app/misc/AmendmentTable.h

```diff
@@ -125,8 +125,10 @@ class AmendmentTable
 
     virtual void addInitial () = 0;
 
-    virtual AmendmentState* addKnown (const char* amendmentID,
-        const char* friendlyName, bool veto) = 0;
+    virtual bool addKnown (uint256 const& amendmentID,
```

Comment:
Comments, comments!  What does this do?  Particularly, what does that inscrutable `bool` return do?